easybuild/easyconfigs/a/ACTC/ACTC-1.1-intel-2016b.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'ConfigureMake'
+
+name = 'ACTC'
+version = '1.1'
+
+homepage = 'https://sourceforge.net/projects/actc'
+description = "ACTC converts independent triangles into triangle strips or fans."
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+skipsteps = ['configure']
+
+buildopts = 'CC="$CC" CFLAGS="$CFLAGS"'
+
+installopts = "INCDEST=%(installdir)s/include/ac/ LIBDEST=%(installdir)s/lib && "
```

Comment:
if the software supports a `make install`, I prefer to use it (even though it's broken in this case since the header file gets copied to `/lib` >_<)